lakefs_provider/__init__.py

```diff
@@ -6,9 +6,11 @@ def get_provider_info():
         "description": "An Airflow provider of lakeFS",
         "hook-class-names": ["lakefs_provider.hooks.lakefs_hook.LakeFSHook"],
         "connection-types": [{
-            'connection-type': 'lakefs',
+            'connection-type': 'LakeFS',
```

Comment:
I think the name is officially "lakeFS", so
```suggestion
            'connection-type': 'lakeFS',
```

Will these forms (with capitals) work for users who say just "lakefs"?  Because that's the most important form.

Reply:
Will change this to lakeFS. It is just the form type in the drop down of the form.